tools/trace_from_ser_port.py

```diff
@@ -1,4 +1,18 @@
+"""Start a live exeuction trace for an embedded target using the selected
+serial port.
 
+You must provide the map file and SVD information needed for translating trace
+values on the command line. You must also specify the serial device. This will
+be of the form 'COMn' on Windows or '/dev/ttyn' on Mac or Linux.
```

Comment:
I have existing code to prompt the user for this info with the active serial ports in a list if they fail to provide it. Remind me to send or add to an issue here.

Reply:
That would be nice.  Is it free to use for open-source projects?